Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic_README.md

```diff
@@ -6,34 +6,33 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 
-* Detonate File - ThreatGrid
-* Detonate File - HybridAnalysis
 * ATD - Detonate File
-* Detonate File - FireEye AX
-* Detonate File - VMRay
-* Detonate File - JoeSecurity V2
-* WildFire - Detonate file
+* Detonate File - Lastline v2
 * Detonate File - ANYRUN
-* Detonate File - Group-IB TDS Polygon
-* Detonate File - Cuckoo
 * CrowdStrike Falcon Sandbox - Detonate file
-* Detonate File - CrowdStrike Falcon Intelligence Sandbox
 * Detonate File - SecneurX Analysis
-* Detonate File - Lastline v2
+* Detonate File - HybridAnalysis
+* Detonate File - CrowdStrike Falcon Intelligence Sandbox
+* Detonate File - Cuckoo
+* Detonate File - FireEye AX
+* Detonate File - VMRay
+* WildFire - Detonate file
+* Detonate File - JoeSecurity V2
+* Detonate File - ThreatGrid
 * Detonate File - JoeSecurity
-* Detonate File - SNDBOX
+* Detonate File - Group-IB TDS Polygon
```

Comment:
whats this?

Reply:
sdk generate docs just changed the order, it didnt add or remove anything besides SNDBX